lxa_iobus/network.py

```diff
@@ -35,6 +36,11 @@ class LxaShutdown(Exception):
 class LxaNetwork:
     node_drivers = []
 
+    class LssStates(enum.Enum):
+        """States of the LSS subsystem"""
+        idle = "Idle"
```

Comment:
This should be upper case

Reply:
Totally right. I've pushed and update that fixes this.